Resources/Prototypes/_NF/Procedural/scrap_vgroid.yml

```diff
@@ -144,6 +144,13 @@
       entity: SpawnMobRogueScapT1
       count: 25
       minGroupSize: 1
+      maxGroupSize: 2
```

Comment:
MFW 50 Rogue

Reply:
65 mobs total with the drones below - might be a considerable problem given the limit of 128 AIs at once - we could increase it and see if it chugs.

Reply:
Kinda,  forgor to change numbers when copied the lines